lib/Filterview.dart

```diff
@@ -35,28 +34,25 @@ class _filterimageState extends State<filter_image> {
     var image = imageLib.decodeImage(imageFile.readAsBytesSync());
     image = imageLib.copyResize(image, width: 600);
     Map imagefile = await Navigator.push(
-      context,
+        context,
         PageRouteBuilder(
-          pageBuilder: (c, a1, a2) =>
-              new PhotoFilterSelector(
-                appBarColor: appBarColor,
-                title: Text("Apply Filter"),
-                image: image,
-                filters: presetFiltersList,
-                filename: fileName,
-                loader: Center(
-                    child: CircularProgressIndicator(
-                  backgroundColor: Colors.teal,
-                  strokeWidth: 2,
-                )),
-                fit: BoxFit.contain,
-              ),
+          pageBuilder: (c, a1, a2) => new PhotoFilterSelector(
+            appBarColor: appBarColor,
+            title: Text("Apply Filter"),
+            image: image,
+            filters: presetFiltersList,
+            filename: fileName,
+            loader: Center(
+                child: CircularProgressIndicator(
+              backgroundColor: Colors.teal,
+              strokeWidth: 2,
+            )),
+            fit: BoxFit.contain,
+          ),
           transitionsBuilder: (c, anim, a2, child) =>
-              FadeTransition(
-                  opacity: anim, child: child),
+              FadeTransition(opacity: anim, child: child),
           transitionDuration: Duration(milliseconds: 1000),
-        )
-    );
+        ));
```

Comment:
change `));` to `),);` and save it. It will reformat it. Probably, there are some more places where same thing is repeated. Make sure to check that too.